docs-partials/builder/azure/arm/Config-not-required.mdx

```diff
@@ -24,7 +24,8 @@
       "resource_group": "ResourceGroup",
       "gallery_name": "GalleryName",
       "image_name": "ImageName",
-      "image_version": "1.0.0"
+      "image_version": "1.0.0",
+      "communityGallery_image_id": "/CommunityGalleries/{cg}/Images/{}/Versions/{}"
```

Comment:
Given that commit organization is good in this PR (well done!), this should perhaps be put in the commit including community galleries support.